src/controllers/Notifications.controller.ts

```diff
@@ -11,6 +11,9 @@ import { NotificationAttributes } from "../interfaces";
 import { verifyToken } from "../utils";
 import User from "../database/models/User.model";
 
+const B_ACCESSKEY = process.env.B_ACCESSKEY as string;
+const S_ACCESSKEY = process.env.S_ACCESSKEY as string;
```

Comment:
```suggestion
import { B_ACCESSKEY, S_ACCESSKEY } from "../utils"
```
create a role constants files in utils and declare the role constants there instead of storing them in your .env